README.md

```diff
@@ -18,20 +132,9 @@
 
 ## 🤝 Contributing
 
+
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
```

Comment:
- [ ] OPTIONAL
It's recommended as good practice that you add an MIT license or any other similar license to your ReadMe for this project.